.expeditor/update_coverage.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eou pipefail
+
+export COVERAGE_DATE
+COVERAGE_DATE="$(date +"%Y-%m-%dT%H:%M:%S")"
+
+# TODO @afiune How can we download artifacts from previous buildkite builds?
+# use instead the agent
+#buildkite-agent artifact download "coverage/*" --build XXX-XXX-XXX(?)
+hab studio run "source .studiorc && code_coverage"
+
+aws --profile chef-cd s3 sync coverage "s3://chef-workstation-coverage/$BUILDKITE_PIPELINE_SLUG/current"
```

Comment:
What is an example of `BUILDKITE_PIPELINE_SLUG`? I want to make sure we are able to differentiate between the code coverage of `chef-analyze` and some (theoretical) other project that uploads coverage

Reply:
`BUILDKITE_PIPELINE_SLUG="chef-chef-analyze-master-verify"`

You can see all the environment variables on the last tab of each buildkite job:
![image](https://user-images.githubusercontent.com/5712253/67972205-d4e59e00-fbd3-11e9-9e98-53ecabda0078.png)

https://buildkite.com/chef/chef-chef-analyze-master-verify/builds/127#c9575ee5-0d46-4b9b-ada5-51b31007013c